privacy-policy.md

```diff
@@ -0,0 +1,73 @@
+
+Private Messages and Non-Public Communications: We provide certain features that allow you to communicate more privately. For example, you can use Private Messages to have private conversations with other Slaent users. When you privately communicate with others through our Services, such as by sending and receiving Private Messages, we will store and process your communications, and information related to them. While features like Private Messages are intended to be private, please remember that recipients may copy, store, and re-share the contents of your communications.
+
+Location Information: We may receive information about your location. For example, you may choose to publish your location in your Posts and in your Slaent profile. We may also determine location by using other data from your device, such as precise location information from GPS, information about wireless networks or cell towers near your mobile device, or your IP address. We may use and store information about your location to provide features of our Services, such as Posting with your location, and to improve and customize the Services.
+
+Links: We may keep track of how you interact with links across our Services, including our email notifications, third-party services, and client applications, by redirecting clicks or through other means. We do this to help improve our Services, to provide more relevant advertising, and to be able to share aggregate click statistics such as how many times a particular link was clicked on. 
+
+Cookies: Like many websites, we use cookies and similar technologies to collect additional website usage data and to improve our Services, but we do not require cookies for many parts of our Services such as searching and looking at public user profiles. A cookie is a small data file that is transferred to your computer or mobile device. We may use both session cookies and persistent cookies to better understand how you interact with our Services, to monitor aggregate usage by our users and web traffic routing on our Services, and to customize and improve our Services. Although most web browsers automatically accept cookies, some browsers’ settings can be modified to decline cookies or alert you when a website is attempting to place a cookie on your computer. However, some Services may not function properly if you disable cookies. We honor the Do Not Track browser option to give you control over how your website visits are used to personalize your Slaent experience and ads.  (#Do Not Track settings? #)
```

Comment:
I don't think Do Not Track is something we can set, iirc it's a per-browser feature.